packages/devtools_app/lib/src/standalone_ui/vs_code/flutter_panel.dart

```diff
@@ -64,52 +66,59 @@ class _VsCodeConnectedPanelState extends State<_VsCodeConnectedPanel> {
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      children: [
-        const SizedBox(height: defaultSpacing),
-        if (widget.api.capabilities.executeCommand)
-          ElevatedButton(
-            onPressed: () =>
-                unawaited(widget.api.executeCommand('flutter.createProject')),
-            child: const Text('New Flutter Project'),
-          ),
-        if (widget.api.capabilities.selectDevice)
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
```

Comment:
use the `denseSpacing` constant from devtools_app_shared/ui.dart

Reply:
Done!